src/Sane/Html.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Kirby\Sane;
+
+use Kirby\Exception\Exception;
+
+/**
+ * Sane handler for HTML files
+ * @since 3.6.0
+ *
+ * @package   Kirby Sane
+ * @author    Bastian Allgeier <bastian@getkirby.com>,
+ *            Lukas Bestle <lukas@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier GmbH
+ * @license   https://opensource.org/licenses/MIT
+ */
+class Html extends Handler
+{
+    /**
+     * HTML sanitization
+     *
+     * @param string $string
+     * @return string
+     */
+    public static function tidy(string $string): string
```

Comment:
It's nit picking, but wouldn't `sanitize()` be the more fitting name next to `validate()`?

Reply:
`Sane::sanitize()` sounds a bit duplicated though. Another alternative: `purify()`. But I'd be happy with either one of these three.

Reply:
But it would be `Sane\Html::sanitize()`. Which I think reads actually quite nice. But if you both prefer `tidy` it's fine. Was just a thought.

Reply:
I had sanitize first and then went for tidy, exactly for that reason. `Sane\Html::sanitize()` would be nice indeed, but we never use it that way.

Reply:
Now that I think about it again, I do like `sanitize()`. The name is more clear as `tidy` could also be just an indentation fixer or minifier. `sanitize` has a clear connection to security.

Reply:
TBH I don't mind enough :) Both names work.

Reply:
https://github.com/getkirby/kirby/pull/3524